### PR TITLE
Add tags in image single result page

### LINKED ISSUE #742: Adding tags in image single result page
## Problem
As mentioned in #682 by @zackkrida, the image single result missed the tag information that is displayed on the audio version. We need to keep consistency on what information and how we display it.

## Description
Add the tag pills in the **Image information** section as [the mockups](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=133%3A6869) show.

### Desktop

CleanShot 2022-01-27 at 17 52 10@2x

### Mobile

CleanShot 2022-01-27 at 17 52 29@2x

## Additional context
1. Tag pill is not a link.
2. The component has two sizes, one for desktop and one for mobile.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #742 by @panchovm

## Description

- Updates the component name with the `V` prefix and its references and changes the default HTML tag to `span`as indicated in the code comment
- Applies text styles of `caption bold` to tags
- Adjust horizontal padding on desktop

## Testing Instructions

Run the project and see the details page of any image with tags, observe if styles matches Figma mockups [[mobile](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=958%3A19663), [desktop](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=1228%3A21557)]

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VImageDetails/VImageDetails.vue [hunk 1]
```diff
@@ -6,6 +6,14 @@
       </h3>
       <VContentReportPopover :media="image" />
     </div>
+    <ul class="flex flex-wrap gap-2 mb-6">
```
[the remark this answers is not in the document]
Comment (krysal) [reply src/components/VImageDetails/VImageDetails.vue:9]: Didn't notice that, thanks!

**Comment by zackkrida** [comment]
Francisco has suggested that these tags be smaller on desktop, and you can see this version here: https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=958%3A18805

Also, it might be better to use `leading-none` and exclusively use padding to determine the height of the tags, it could give you more control!

**Comment by panchovm** [comment]
It looks great ✨ Nice work @krysal

**Comment by panchovm** [comment]
The versions I am seeing are the correct ones @zackkrida. `large` on desktop and `small` on mobile.

**Review by zackkrida — APPROVED** [review]
Looks perfect!

**Review by obulat — APPROVED** [review]
Looks great!
The testing instructions with the link to the specific Figma mockups are 🔥 
I wonder if we could also remove the `PhotoTags.vue` in this PR, or should it be a different small PR?

**Review by panchovm — APPROVED** [review]
Excellent work

**Comment by krysal** [comment]
@obulat Already done in #872 🧹
Thanks for reviewing!

**Pull request merged**